### ISSUE TITLE
Enums were all nullable - 2.11.13

### ISSUE THREAD

**Issue by anandsubbu007** [epam-cross-platform-lab/swagger-dart-code-generator#667]
**Describe the bug**
when an enum is defined as required and nullable is false. it is still generating nullable variable

**To Reproduce**
```
	"type": {
		"default": "EnumType",
		"nullable": false,
		"$ref": "#/components/schemas/EnumType"
	},
----------------------------
"required": [
	"type",
	"lastUpdateOn",
	"isDeleted"
]
---------------------------
	"EnumType": {
		"type": "string",
		"enum": [
			"Type1",
			"Type2"
		]
	},
```
**Expected behaviour**
Enum that defined required is true or nullable is false must be not nullable

**Library version used:**
^2.11.13

**Comment by Vovanella95** [comment]
Hi @anandsubbu007 , indeed! Sorry for this bug))) Will be fixed soon!

**Comment by Vovanella95** [comment]
Hi @anandsubbu007 , should be fixed in `2.12.1`
